client/app/bundles/course/user-invitations/components/forms/IndividualInvitations.tsx

```diff
@@ -54,8 +78,70 @@ const IndividualInvitations: FC<Props> = (props) => {
     });
   };
 
+  const appendInputs = (results: InvitationEntry[]): void => {
+    const lastRow = fields[fields.length - 1];
+
+    if (fields.length === 1 && results.length > 0) {
+      remove(0);
+    }
+
+    results.forEach((entry) => {
+      append({
+        name: entry.name,
+        email: entry.email,
+        role: lastRow.role,
+        phantom: lastRow.phantom,
+        ...(permissions.canManagePersonalTimes && {
+          timelineAlgorithm: lastRow.timelineAlgorithm,
+        }),
+      });
+    });
+  };
+
+  const parsedEmail = parseEmails(nameEmailInput);
+
   return (
     <>
+      <div className="w-full flex justify-between text-center items-center">
+        <TextField
+          className="w-[130rem]"
+          hiddenLabel
+          multiline
+          name="nameEmailInvitation"
+          onChange={(e): void => setNameEmailInput(e.target.value)}
+          placeholder={t(translations.nameEmailInput)}
+          size="small"
+          value={nameEmailInput}
+          variant="filled"
+        />
+        <Button
+          className="w-1/8 h-[3.5rem]"
+          color="primary"
+          onClick={(): void => {
+            appendInputs(parsedEmail.results);
+            setNameEmailInput(
+              parsedEmail.errors.length > 0
+                ? parsedEmail.errors.join('; ')
+                : '',
```

Comment:
If the length is 0 then `parsedEmail.errors.join('; ')` will still be an empty string, what's the point of this ternary if-else then?

Reply:
done